webrtc-stats.html

```diff
@@ -1064,6 +1144,57 @@
           </section>
         </div>
       </section>
+      <section id="remoteoutboundrtpstats-dict*">
+        <h3>
+          RTCRemoteOutboundRTPStreamStats dictionary
+        </h3>
+        <p>
+          The <dfn>RTCRemoteOutboundRTPStreamStats</dfn> dictionary represents
+          the remote endpoint's measurement metrics for its outgoing RTP stream
+          (our incoming RTP stream).
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCRemoteOutboundRTPStreamStats : RTCSentRTPStreamStats {
+             DOMString           localId;
+             DOMHighResTimeStamp remoteTimestamp;
```

Comment:
This is a separate issue, but wouldn't it make sense to have a remoteTimestamp on RTCRemoteInboundRTPStreamStats as well?